migrate_pim/apply_patch/apply_patch_ee_flexibility.rst

```diff
@@ -1,21 +1,21 @@
 How to apply a patch - Enterprise Edition - Flexibility Cloud offer
 ========================================================================
 
-In the following example, Akeneo PIM version 6.0.2 has just been released and we are using an Akeneo PIM version 6.0.1.
+In the following example, Akeneo PIM version 6.0.50 has just been released and we are still using an Akeneo PIM version 6.0.1.
```

Comment:
```suggestion
In the following example, Akeneo PIM version 6.0.50 has just been released and we are still using Akeneo PIM version 6.0.1.
```
Removes the `an` before `Akeneo PIM` because that isn't standard English